### PR TITLE
Added multithreading for 4x quicker applying

### PR DESCRIPTION
**Pull request by BigweldIndustries**
Hey all! I recently came across from r/antiwork and thought I would give this script a try. Thanks to the awesome work of SeanDaBlack and other contributors, I found it does. However, as the selenium snob I am, I felt it had a bit to be desired. First of all, it is now supported on linux, and requires no webdriver installation. This was done via a pip dependency called ``webdriver_manager``. I then sped up the program to send about three at a time (not too many, chrome windows can be laggy). This was done through the ``futures`` dependency, which allows the simple creation of threadpools. Best of luck, comrades!

### CODE REVIEW

**Comment by tabrown2000** [comment]
Nicely done. I did a similar thing with my fork, 4 threads at a time for a few hours - 1500+ applications. Now I'm wishing I'd used futures instead of multiprocessing - looks a lot cleaner. Too bad the reCaptcha ruined my fun

**Comment by BigweldIndustries** [comment]
Yes, futures is incredible. Makes multithreading so much easier, and isn't that hard to implement. Whenever manually working with threads outside of threadpools my brain just dies.

File: README.md [hunk 1]
```diff
@@ -4,30 +4,8 @@ Credit to SeanDaBlack for the basis of the script.
 req.py is selenium python bot.
 sc.js is a the base of the ios shortcut [COMING SOON]
 
-# Setup
-
-On mac/pc:
+# Setup (Mac/Windows/Linux)
```
Comment (pws1453) [inline README.md:7]: This is great that you've been able to streamline the process for chromedriver as such. Code related to this from this PR should be merged to main immediately.

File: req.py [hunk 2]
```diff
@@ -176,159 +213,215 @@ def random_email(name=None):
...
+        driver.close()
 
-        try:
-            fill_out_application_and_submit(driver, rand_num)
-        except Exception as e:
-            print(f"failed to fill out app and submit: {str(e)}")
-            driver.close()
-            continue
+def main():
+    global processes
```
Comment (pws1453) [inline req.py:413]: This is the proper way to run multiple things simultaneously. This functionality mirrors what I believe is the intent of https://github.com/SeanDaBlack/KelloggBot/pull/16 , but uses the proper library.

**Review by pws1453 — APPROVED** [review]
Code is written well, and is readable. Functionality improves tremendously. Can be pulled into main, but preferably merged with the readability changes made in https://github.com/SeanDaBlack/KelloggBot/pull/26

File: README.md (hunk 1 above)
Comment (BigweldIndustries) [reply README.md:7]: I created a seperate pull request due to this one becoming a bit buggy due to many different purposes.

**Comment by BigweldIndustries** [comment]
Sorry all, this is behind many commits and I am meaning to rework lots of this. I have created a seperate request for webdriver-manager. I will create one later for multiproccessing after captcha and webdriver-manager are implemented. For now, defer to #27

**Pull request closed without merging**